lib/bumper.js

```diff
@@ -12,14 +21,48 @@ const utils = require('./utils')
  * @class
  */
 class Bumper {
+  // ====================================================================================
+  // Public Methods
+  // ====================================================================================
 
   /**
-   * @param {Vcs} vcs - the vcs instance to use
-   * @param {Config} config - the bumper config object
+   * @param {Object} params - params obj
+   * @param {Config} params.config - the bumper config object
+   * @param {Vcs} params.vcs - the vcs instance to use
+   * @param {Ci} params.ci - the ci instance to use
    */
-  constructor (vcs, config) {
-    this.vcs = vcs
-    this.config = config
+  constructor (params) {
+    this.config = params.config
+    this.vcs = params.vcs
+    this.ci = params.ci
```

Comment:
Since you are already using lodash in this file why not do something like:
```js
const filteredParams = __.pick(params, ['ci', 'config', 'vcs'])
__.assign(this, filteredParams)
```